inksdk/ink/src/main/java/com/microsoft/device/ink/InkView.kt

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License.
+ */
+
+package com.microsoft.device.ink
+
+import android.content.Context
+import android.graphics.*
```

Comment:
Instead using the `*` char adding all dependencies from that package, please, use the specific dependencies you are going to use. (ktlint analysis will complain too).